pkg/scm/git/git.go

```diff
@@ -196,8 +196,24 @@ func ParseURL(source string) (*url.URL, error) {
 }
 
 func useCopy(protoSpecified bool, source string) bool {
-	glog.V(4).Infof("useCopy proto spec %v local git repo %v has git bin %v", protoSpecified, isLocalGitRepository(source), hasGitBinary())
-	return !isLocalGitRepository(source) || !hasGitBinary()
+	isEmpty := isEmptyGitRepository(source)
+	isLocal := isLocalGitRepository(source)
+	hasGit := hasGitBinary()
```

Comment:
Would it be OK if I got rid of that glog, or split it up?  I'll put something up